deploy/iso/minikube-iso/arch/x86_64/package/hyperv-daemons/hyperv-daemons.mk

```diff
@@ -5,7 +5,7 @@
 ################################################################################
 
 HYPERV_DAEMONS_VERSION = $(call qstrip,$(BR2_LINUX_KERNEL_VERSION))
-HYPERV_DAEMONS_SITE = https://www.kernel.org/pub/linux/kernel/v5.x
+HYPERV_DAEMONS_SITE = https://www.kernel.org/pub/linux/kernel/v6.x
```

Comment:
When I tried to build 6.12 kernel few weeks ago I could not compile it because the sources were not found in this url. Maybe this was fixed recently?